monai/apps/auto3dseg/ensemble_builder.py

```diff
@@ -467,6 +470,8 @@ def _pop_kwargs_to_get_image_save_transform(self, **kwargs):
             "resample": kwargs.pop("resample", False),
             "print_log": False,
             "savepath_in_metadict": True,
+            "data_root_dir": kwargs.pop("data_root_dir", data_root_dir),
+            "separate_folder": kwargs.pop("separate_folder", False),
```

Comment:
Thanks, @mingxin-zheng perhaps we can enhance one of the tests to make sure this default behaviour, suggestions?

Reply:
@wyli just opened up a ticket to track here:
https://github.com/Project-MONAI/MONAI/issues/6482